2-projects/1-todo-list/script.js

```diff
@@ -1,6 +1,50 @@
 function populateTodoList(todos) {
   let list = document.getElementById("todo-list");
   // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+  while (list.hasChildNodes()) {
+    list.removeChild(list.firstChild);
+  }
+
+  todos.forEach((element) => {
+    const listItem = document.createElement("li"); 
+    listItem.style.backgroundColor = "white";
+    listItem.style.border = "1px solid grey";
+    listItem.style.width = "550px";
+
+    const taskText = document.createElement("p");
+    taskText.innerText = element.task;
+    listItem.append(taskText);
+
+    const spanEl = document.createElement("span");
+    let spanClasses = ["badge", "bg-primary", "rounded-pill"];
+    spanEl.classList.add(...spanClasses);
+
+    const checkButton = document.createElement("i");
+    const trashButton = document.createElement("i");
+    const checkClasses = ["fa", "fa-check"];
+    const trashClasses = ["fa", "fa-trash"];
+    checkButton.classList.add(...checkClasses);
+    trashButton.classList.add(...trashClasses);
+
+    spanEl.append(checkButton, trashButton);
+    listItem.append(spanEl);
+
+    list.append(listItem);
+
+    checkButton.addEventListener("click", () => {
+      if (listItem.style.textDecoration == "") {
+        listItem.style.textDecoration = "line-through";
+        element.completed = true;
+      } else {
+        listItem.style.textDecoration = "";
+        element.completed = false;
+      }
+    });
+
+    trashButton.addEventListener("click", () => {
+      listItem.remove();
+    });
+  });
```

Comment:
I would suggest breaking this function down into smaller functions so it's easier to read, e.g.
* function to toggle strikethrough can be extracted
* function for generating a buttons (checkButton, trashbutton), since you have some repeated code and is completely unrelated to to-do list rendering. This can accept a parameter for icon class ("fa-trash", ..)